File: messages/src/main/java/com/thundashop/core/resturantmanager/PaymentTransaction.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.thundashop.core.resturantmanager;

import com.thundashop.core.common.DataCommon;
import java.util.ArrayList;
import java.util.Date;
import java.util.List;

/**
 *
 * @author ktonder
 */
public class PaymentTransaction extends DataCommon {
    public String status = "started";
    public String userId = "";
    public List<ResturantCartItem> cartItems = new ArrayList();
    public String orderId = "";
    
}

File: html5apps/terminal/events/core/resturantmanager/PaymentTransaction.php
<?php
class core_resturantmanager_PaymentTransaction extends core_common_DataCommon  {
	/** @var String */
	public $status;

	/** @var String */
	public $userId;

	/** @var core_resturantmanager_ResturantCartItem[] */
	public $cartItems;

	/** @var String */
	public $orderId;

}
?>